core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -2232,9 +2266,11 @@ CacheConfigurationException storeConfiguredHasBothReadAndWriteOnly(String storeC
    EncodingException invalidJson(String s);
 
    @Message(value = "The backup '%s' configuration 'failure-policy=%s' is not valid with an ASYNC backup strategy.", id = 656)
+   @Description("Need more info.")
```

Comment:
@pruivo Are there any failure policies that are compatible with async backups?

Reply:
yes. WARN and IGNORE are valid policies

Reply:
WARN -> prints a message in the logs 
IGNORE -> do not print anything 
In any case, the update is retried in the background